### PR TITLE
blog: july newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
I'm running late this month, but here it is for initial drafts. I'm going to add a section on the birthday parties so far with a few photos tomorrow, but reviews of the rest (plus whatever I missed) are very welcome.

### CODE REVIEW

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Our 7th birthday! Also, Foreman 1.12 released, 1.11.4, Katello 3.1 in RC,
+  Red Hat Summit roundup, plugin releases, ... a busy month!
+---
+
+Time again for the Foreman Newsletter - slightly late perhaps, but definitely
```
Comment (ohadlevy) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: not sure if anyone actually cares about the publish date :)

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+### Foreman's 7th Birthday!
+
+On July 13th, Foreman turned 7 years old! Parties have been happening in some
+of our communities (see the [birthday events][birthday] page) and there's still
+a few to come.
+
+[birthday]: https://theforeman.org/2016/06/foremans-7th-birthday-events.html
+
+7 years is quite a milestone - and we're still growing! We'd like to thank
```
Comment (dLobatog) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: "we're still growing" [citation needed] :smile: 

Not challenging it, I just don't know if that's true or not!

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+Time again for the Foreman Newsletter - slightly late perhaps, but definitely
+not forgotten! It's been a busy month, and all the details are coming right up.
+
+### Foreman's 7th Birthday!
+
+On July 13th, Foreman turned 7 years old! Parties have been happening in some
+of our communities (see the [birthday events][birthday] page) and there's still
+a few to come.
+
+[birthday]: https://theforeman.org/2016/06/foremans-7th-birthday-events.html
```
Comment (dLobatog) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: pics?

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+Katello 3.1 is also in the Release Candidtae phase, and as ever, your testing
+is greatly valued. If you'd like to help with testing, you can get involved
+[here][31rc1]. Thanks to all our awesome testers!
+
+[31rc1]: https://groups.google.com/d/topic/foreman-users/ARgW31tXj7w/discussion
+
+### Foreman Ansible 1.0, now with support for roles
+
+Ansible is a big topic in the community right now, so it's great news that
+Daniel has released Foreman Ansible 1.0 with support for Ansible roles! He's
```
Comment (dLobatog) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: Maybe have my name link to my github profile or say dlobatog,  I doubt readers will know who's exactly 'Daniel'

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+may want to take a look at the [migration page][p3to4] we've put together to
+document the upgrade steps. This has been confirmed working by a few people,
+but as with all complex upgrades, do take care.
+
+[p3to4]: http://projects.theforeman.org/projects/foreman/wiki/Upgrading_from_Puppet_3_to_4
+
+### 1.11.4 bugfix release
+
+The final release in our 1.11 series, 1.11.4 was released on July 25th. This
+brings 3 CVE fixes as well as numerous bug fixes. It's also likely to be the
```
Comment (dLobatog) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: Link here to the CVEs in particular, I'd be interested if I was an user

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md (hunk 2 above)
Comment (GregSutcliffe) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: Which of many vanity metrics would you like? :p Downloads, Redmine accounts, mailing list membership, Twitter followers, etc - all increased over the last year.

I know, I'll do a 1-year-of-community-management blog post in a month's time :)

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md (hunk 3 above)
Comment (GregSutcliffe) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: Good idea, incoming

File: _posts/2016-07-26-foreman-community-newsletter-july-2016.md (hunk 4 above)
Comment (GregSutcliffe) [inline _posts/2016-07-26-foreman-community-newsletter-july-2016.md]: Whoops, good catch

**Comment by GregSutcliffe** [comment]
@dLobatog thanks. Is there a good way to show the pictures 2x2 ?

**Comment by GregSutcliffe** [comment]
@dLobatog (or anyone :P) - last day of July today. Updated to reflect the demo is now in the past. Is this good to go?

**Comment by dLobatog** [comment]
Yeah let's publish this

**Comment by dLobatog** [comment]
Merged as e63d12e7fd1336530bf646375de43519bbe5c2a4, thanks @GregSutcliffe!

**Pull request closed without merging**